### PR TITLE
Fixes part of #2210: Changes in Administrator Controls[a11y]

### LINKED ISSUE #2210: [A11y] Changes in Admin Controls Download Permissions View
**Describe the bug**

Admin Control Download Permission View is not fully compatible with some users from the Accessibility point of view.
Screenshot_2020-12-02-11-13-39-64_80a9469f53bfe75dbc6e8989be28e2f0
Issue 1 - The Size of Switch button must be larger ( minimum 48dpX48dp) 
Issue 2 - Multiple items have the same Description as that of Switch Button
Issue 3 - The Contrast ratio of the text (foreground and background) inside the TextViews is 2.68 but the required one 4.50 

**To Reproduce**
Steps to reproduce the behaviour:
1. Start the Accessibility Scanner App on your device
2. Open the Oppia Android App and scan the screenshots with the Accessibility Scanner
3. You will see the tests that are not passed by that screen
4. Repeat this for other screens

**Expected behaviour**
All the tests must be passed by scanning the screenshots for both Tablet and Mobile Devices in every orientation.

**Screenshots**

| Issue 1  |    Issue 2    |  Issue 3 |
|:--------:|:-------------:|:---------:|
|Screenshot_2020-12-02-11-13-18-10_80a9469f53bfe75dbc6e8989be28e2f0|Screenshot_2020-12-02-11-13-28-16_80a9469f53bfe75dbc6e8989be28e2f0 | Screenshot_2020-12-02-11-13-04-76_80a9469f53bfe75dbc6e8989be28e2f0 |  

**Device**
- Real Device - RealMe3
- SDK version - 29

**Additional context**
The Screenshots are descriptive enough for understanding the issues but still, if there is a need of clarification then you can comment here. Multiple Pull Requests can also be made for solving these issues.

Reference - [https://github.com/oppia/oppia-android/wiki/Working-on-UI#accessibility-scanner](https://github.com/oppia/oppia-android/wiki/Working-on-UI#accessibility-scanner)

### PR DESCRIPTION
**Pull request by Arjupta**
## Explanation

Fixes part of #2210,
(Issue 3) Changed the text color as per Chantel's suggestion

|Before|After|Tests|
|:-:|:-:|:-:|
|Screenshot_1611495503 | Screenshot_1611495542 | Screenshot_1611495550|

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: app/src/main/res/layout/administrator_controls_download_permissions_view.xml [hunk 1]
```diff
@@ -119,7 +119,7 @@
         style="@style/Subtitle2"
         android:background="@color/white"
         android:text="@string/administrator_controls_auto_update_message"
-        android:textColor="@color/divider"
+        android:textColor="#555555"
```
Comment (rt4914) [inline app/src/main/res/layout/administrator_controls_download_permissions_view.xml]: Use this here too: `accessible_light_grey `

**Review by rt4914 — CHANGES_REQUESTED** [review]
@ARJUPTA Please add before and after screenshot along with the scanner result screenshot (even if it is successful).

File: app/src/main/res/layout/administrator_controls_download_permissions_view.xml (hunk 1 above)
Comment (Arjupta) [reply app/src/main/res/layout/administrator_controls_download_permissions_view.xml]: done

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

**Pull request merged**